.github/workflows/rolling-coverage-build.yml

```
@@ -30,7 +30,7 @@ on:
 
 jobs:
   coverage_rolling:
-    uses: ros-controls/ros2_control_ci/.github/workflows/reusable-build-coverage.yml@master
+    uses: ros-controls/ros2_control_ci/.github/workflows/reusable-build-coverage.yml@coveragepy
```

```suggestion
    uses: ros-controls/ros2_control_ci/.github/workflows/reusable-build-coverage.yml@master
```
once https://github.com/ros-controls/ros2_control_ci/pull/115 is merged